src/ingestion_data.py

```diff
@@ -4,6 +4,15 @@
 
 from src.graph_model import (Version, EventType, SecurityEvent, FeedBackType, StatusType, EcosystemType)
 from src.parse_datetime import from_date_str, get_date, get_year, get_yearmonth
+from src.config import MAX_STRING_LENGTH
+
+CVE_REGULAR_EXPRESSION = r"CVE-\d{4}-\d{4,}"
+
+
+def get_cves_from_text(data: str) -> set:
+    """Get the CVEs from the given text."""
+    cves = re.findall(CVE_REGULAR_EXPRESSION, data.upper())
+    return set(cves)
```

Comment:
Return it as a list and a set can be formulated from the list from the caller side.

Reply:
done